packages/api/src/schemas/envConfig.json

```diff
@@ -6,6 +6,39 @@
     "$schema": {
       "type": "string"
     },
+    "auth": {
+      "type": "object",
+      "description": "Existing tab AAD app configuration.",
+      "properties": {
```

Comment:
@adashen @KennethBWSong Is this reasonable/accurate if we call it **tab** AAD app here?

Reply:
Discussed with @blackchoey and @adashen, we should not call it tab AAD app since it will also support SSO for bot and messaging extension.

Reply:
@KennethBWSong So what's your suggestion here to provide an accurate description for this AAD app?

Reply:
Offline synced with @KennethBWSong , updated to "Existing AAD app configuration."